sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -416,6 +416,14 @@ object SQLConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefaultString("10MB")
 
+  val SHUFFLE_HASH_JOIN_FACTOR = buildConf("spark.sql.shuffleHashJoinFactor")
```

Comment:
nit: how about `spark.sql.shuffledHashJoinFactor`? We adhere to shuffle`d` hash join in config name, e.g. existing one - `spark.sql.adaptive.maxShuffledHashJoinLocalMapThreshold`